modules/javafx.graphics/src/main/java/javafx/print/PrinterJob.java

```diff
@@ -243,9 +223,20 @@ public synchronized Printer getPrinter() {
      * <p>
      * Setting a null value for printer will install the default printer.
      * Setting the current printer has no effect.
-     * @param printer to be used for this print job.
+     * @return the <code>Printer</code> for this job
```

Comment:
If we are using {@code Printer} preferred pattern above for new addition, probably we should use here too, no?

Reply:
done